suite-native/atoms/src/Sheet/BottomSheetHeader.tsx

```diff
@@ -9,15 +9,16 @@ import { Box } from '../Box';
 type BottomSheetHeaderProps = {
     title?: string;
     subtitle?: string;
+    showClose: boolean;
```

Comment:
nit: We have an unwritten rule that boolean variables have `is/are/has`etc.  prefix. So the boolean type is apparent just by looking at the name.
```suggestion
    isCloseDisplayed: boolean;
```

Reply:
changed in 6056663